whylog/teacher/rule_validation_problems.py

```diff
@@ -0,0 +1,72 @@
+from collections import namedtuple
+
+ValidationResult = namedtuple('ValidationResult', ['errors', 'warnings'])
+
+
+class ValidationResult(object):
+    def __init__(self, errors, warnings):
+        """
+        :param errors: problems preventing form rule saving
+        :param warnings: problems accepted while rule saving
+        :type errors: [RuleValidationProblem]
+        :type warnings: [RuleValidationProblem]
+        """
+        self.errors = errors
+        self.warnings = warnings
+
+    @classmethod
+    def result_from_results(cls, results):
+        errors = sum([result.errors for result in results], [])
+        warnings = sum([result.warnings for result in results], [])
+        return ValidationResult(errors, warnings)
+
+    def acceptable(self):
+        return not self.errors
```

Comment:
Add method:
def get_parser_id_problems(self, line_id):
which returns Problems with given line_id and analogue function for constraint_id.